Sprout/View Controllers/PostDetailsViewController.m

```diff
@@ -0,0 +1,53 @@
+//
+//  PostDetailsViewController.m
+//  Sprout
+//
+//  Created by Sarah Wang on 7/23/21.
+//
+
+#import "PostDetailsViewController.h"
+#import "Post.h"
+#import "DateTools.h"
+
+@interface PostDetailsViewController ()
+
+@property (weak, nonatomic) IBOutlet UIImageView *profileImageView;
+@property (weak, nonatomic) IBOutlet UILabel *nameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *usernameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *timestampLabel;
+@property (weak, nonatomic) IBOutlet UILabel *progressLabel;
+@property (weak, nonatomic) IBOutlet UILabel *captionLabel;
+@property (weak, nonatomic) IBOutlet UIButton *likeButton;
+@property (weak, nonatomic) IBOutlet UIButton *commentButton;
+
+@end
+
+@implementation PostDetailsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    [self refreshData];
+}
+
+- (void)refreshData {
+    PFUser *currUser = self.post.author;
```

Comment:
nit: I think abbreviations are sometimes not helpful unless they're really well known and highly used. This doesn't add much imo.